### DISCUSSION TITLE
Python3 issue since update

### DISCUSSION THREAD

**Discussion by SpidFightFR** [void-linux/void-packages#46523] [Q&A] [answered]
Since the update of python3 (https://github.com/void-linux/void-packages/commit/a3c8698631a7cfb0ad81a4e4ff4a0d22c321f1c6), xbps-src template have a later version than the one available in the official [repos](https://voidlinux.org/packages/?arch=x86_64&q=python3).

It is an issue because for all python-related packages, a rebuild of python3 and deps is required, thus it makes an issue with workflows checks in PR.

Will the python3 package be updated in the repos? :confused:

**Comment by classabbyamp** [comment, marked as answer]
[it's still building](https://build.voidlinux.org/waterfall), just take the weekend off or checkout a commit before that

**Reply by SpidFightFR** [reply]
wow ! Makes sense, it took me like 1 hour and a half just for Python 3 and a few deps for mesa. Alright ! i hope it'll go well ! :sweat_smile: 

EDIT: thanks for your quick answer !
